ResourceStatusSystem/Client/ResourceManagementClient.py

```diff
@@ -139,49 +141,39 @@ def deleteAccountingCache( self, name = None, plotType = None,
         time-stamp from which the result is effective
       **lastCheckTime** - `datetime`
         time-stamp setting last time the result was checked
-      **meta** - `[, dict]`
-        meta-data for the MySQL query. It will be filled automatically with the\
-       `table` key and the proper table name.
 
     :return: S_OK() || S_ERROR()
     '''
-    # Unused argument
-    # pylint: disable=unused-argument
-    return self._query( 'delete', 'AccountingCache', locals() )
+
+    return self.rmsDB.delete( 'AccountingCache', self._prepare(locals()) )
 
 
   # GGUSTicketsCache Methods ...................................................
-  #FIXME: only one method
 
   def selectGGUSTicketsCache( self, gocSite = None, link = None, openTickets = None,
-                              tickets = None, lastCheckTime = None, meta = None ):
-    # Unused argument
-    # pylint: disable=unused-argument
-    return self._query( 'select', 'GGUSTicketsCache', locals() )
+                              tickets = None, lastCheckTime = None ):
+
+    return self.rmsDB.select( 'GGUSTicketsCache', self._prepare(locals()) )
 
 
   def deleteGGUSTicketsCache( self, gocSite = None, link = None, openTickets = None,
-                              tickets = None, lastCheckTime = None, meta = None ):
-    # Unused argument
-    # pylint: disable=unused-argument
-    return self._query( 'delete', 'GGUSTicketsCache', locals() )
+                              tickets = None, lastCheckTime = None ):
+
```

Comment:
I added a docstring but it is incomplete because the method was not created by me. For example I know that the variable `tickets` must be a string but I don't know what is it for in order to write a description.